tests/iostability/test_iostability_degraded_path.py

```diff
@@ -94,6 +95,21 @@ def setup_class(cls):
         resp = s3bench.setup_s3bench()
         assert_utils.assert_true(resp)
         cls.remote_dir_path = cls.test_cfg['remote_path']
+        cls.log.info("Get %s pod to be deleted", POD_NAME_PREFIX)
+        sts_dict = cls.master_node_list[0].get_sts_pods(pod_prefix=POD_NAME_PREFIX)
+        sts_list = list(sts_dict.keys())
+        cls.log.debug("%s Statefulset: %s", POD_NAME_PREFIX, sts_list)
+        sts = secrets.SystemRandom().sample(sts_list, 1)[0]
+        cls.delete_pod = sts_dict[sts][-1]
+        cls.log.info("Pod to be deleted is %s", cls.delete_pod)
+        cls.set_type, cls.set_name = cls.master_node_list[0].get_set_type_name(
+            pod_name=cls.delete_pod)
+        if cls.set_type == STATEFULSET:
+            resp = cls.master_node_list[0].get_num_replicas(cls.set_type, cls.set_name)
+            assert_utils.assert_true(resp[0], resp)
+            cls.num_replica = int(resp[1])
+        else:
```

Comment:
if only for server and data pod, we may no longer need this else as going forward its going to be statefulset only.
Agree? @ApurwaMohite

Reply:
If going forward data and server will be only statefulset then we don't need this if-else. We can by default assume that set_type is statefulset.

Reply:
@priyankaborawake  please change accordingly?